client/experimentalclient/client.go

```diff
@@ -1,4 +1,4 @@
-package experimental
+package experimentalclient
```

Comment:
Just for testing. Can you not change this?
I doubt the init() has some impact that we didn't know.